providers/anthropic/controls_usage_test.go

```
@@ -0,0 +1,87 @@
+		_, _ = io.WriteString(w, clampedControlsAnthropicStream)
+	}))
+	defer server.Close()
+
+	provider := New(WithAPIKey("test-key"), WithEndpoint(server.URL), WithModel(ModelClaudeSonnet45))
+	iterator, err := provider.Stream(context.Background(),
+		llm.WithMessages(llm.NewUserTextMessage("hello")),
+		llm.WithReasoningEffort(llm.ReasoningEffortMedium),
+		llm.WithTemperature(0.7),
+	)
+	assert.NoError(t, err)
+	defer iterator.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `iterator.Close` is not checked

(errcheck)